src/mappers/provinces/province_mapping_importer.cpp

```diff
@@ -18,7 +18,17 @@ mappers::ProvinceMappingImporter::ProvinceMappingImporter()
    parser_.registerKeyword("hoi4", [this](std::istream& theStream) {
       hoi4_provinces_.push_back(commonItems::getInt(theStream));
    });
-   parser_.registerKeyword("comment", commonItems::ignoreItem);
+   parser_.registerKeyword("comment", [this](std::istream& the_stream) {
```

Comment:
Generally, this is a very poor way to assign provinces to regions. The comments are very unreliable after a human hand (mine for example) is done with map alterations, unless you are exclusively relying on automapping. 
Really, the only proper way is to ping regions and area definitions, but leave as is if this is just a quick and dirty patch.

Reply:
The intent is actually to weed all of that out. :-) It got real bad with the map changes in Vic3 1.7.